### PR TITLE
feat: microblock details page

### LINKED ISSUE #303: Microblock details
A new panel should be added with the following:
- Microblock Hash
- Height
- Time
- Transactions count

Something should also tell the user a microblock is inside a keyblock and there's a sequence. Ideally a simple navigation between them would be great.

### LINKED ISSUE #339: Microblock details page

### PR DESCRIPTION
**Pull request by lukeromanowicz**
## Description
Resolves #303 

Navigation to the previous and next microblock has been split into https://github.com/aeternity/aescan/issues/375 since there is still some open discussion on how it is supposed to work.

## Demo
image

image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] My change does not require a change to the documentation.

### CODE REVIEW

**Comment by lukeromanowicz** [comment]
Example: https://pr-377-aescan.stg.aepps.com/microblocks/mh_2dJbtkU6LcRLmEEPdRLqgxmn2q4Q1WQMxNizk3VbVQDmKUCRgU

File: src/components/MicroblockDetailsPanel.vue [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+          <th class="microblock-details-panel__table-header">
+            Transactions Count
+          </th>
+          <td class="microblock-details-panel__data">
+            {{ formatNumber(microblockDetails.transactions_count) }}
+          </td>
+        </tr>
+        <tr class="microblock-details-panel__row">
+          <th class="microblock-details-panel__table-header">
+            API links
```
Comment (janmichek) [inline src/components/MicroblockDetailsPanel.vue:60]:
```suggestion
            API Links
```

**Review by janmichek — APPROVED** [review]
Overall I like it .) Good job. Please consider my suggestions.

File: src/components/MicroblockDetailsPanel.vue (hunk 1 above)
Comment (lukeromanowicz) [reply src/components/MicroblockDetailsPanel.vue]: Good idea, applied here in all other places where it was also wrong

**Comment by CharisSiarampalis** [comment]
@lukeromanowicz Design looks ok. But how can someone navigate to the microblock ?. Searching for it does not work.

image

**Review by CharisSiarampalis — APPROVED** [review]

File: src/components/MicroblockDetailsPanel.vue [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+              </span>
+              <span class="microblock-details-panel__hash-ellipse">
+                {{ formatEllipseHash(microblockDetails.prev_key_hash) }}
+              </span>
+            </app-link>
+          </td>
+        </tr>
+        <tr class="microblock-details-panel__row">
+          <th class="microblock-details-panel__table-header">
+            Microblock Height
```
[the remark this answers is not in the document]
Comment (lukeromanowicz) [reply src/components/MicroblockDetailsPanel.vue]: You're right - fixed

File: src/components/MicroblockDetailsPanel.vue [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+          </th>
+          <td class="microblock-details-panel__data">
+            {{ formatNumber(microblockDetails.height) }}
```
[the remark this answers is not in the document]
Comment (lukeromanowicz) [reply src/components/MicroblockDetailsPanel.vue]: Absolutely, fixed

**Pull request merged**